_src/pl/static/js/checkbox-replace.js

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Is there a source for this script? would be nice to attribute and have a way to track back where it came from.